ql/ql/src/queries/style/AlertMessage.ql

```diff
@@ -0,0 +1,140 @@
+/**
+ * @name Alert messages style violation
+ * @description Alert message that doesn't follow some aspect of the style guide.
```

Comment:
An alert
```suggestion
 * @description An alert message that doesn't follow the style guide is harder for end users to digest.
```

Follows the usual description style: `if ... then ...`. Maybe that is for another query.